### PR TITLE
Cleanup and move to /contributors/guide of pull requests doc

### PR DESCRIPTION
**Pull request by guineveresaenger**
In partial fulfillment of #1514, this doc is aimed at contributors. It needs a parallel doc for reviewers.

This is a work in progress, and lively comments and debate are encouraged.

### CODE REVIEW

File: contributors/devel/pull-requests.md [hunk 1]
```diff
@@ -1,329 +1 @@
...
-
-You might be overruled, but you might also prevail. We're pretty reasonable people. Mostly.
-
-Another phenomenon of open-source projects (where anyone can comment on any issue) is the dog-pile - your PR gets so many comments from so many people it becomes hard to follow. In this situation, you can ask the primary reviewer (assignee) whether they want you to fork a new PR to clear out all the comments. You don't HAVE to fix every issue raised by every person who feels like commenting, but you should answer reasonable comments with an explanation.
-
-## 9. Common Sense and Courtesy
-
-No document can take the place of common sense and good taste. Use your best judgment, while you put
-a bit of thought into how your work can be made easier to review. If you do these things your PRs will get merged with less friction.
+This file has been moved to [/contributors/guide/pull-requests.md](/contributors/guide/pull-requests.md).
```
Comment (cblecker) [inline contributors/devel/pull-requests.md]: Tombstone files should have an end date.

e.g.:
```
<!--
This file is a placeholder to preserve links.  Please remove after 3 months or the release of kubernetes 1.10, whichever comes first.
-->
```

Comment (guineveresaenger) [reply contributors/devel/pull-requests.md]: Noted. It's mainly that all the links in other k/subrepos are in need of updating in the meantime.

File: contributors/guide/pull-requests.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+- [Before You Submit a Pull Request](#before-you-submit-a-pr)
+  * [Run Local Verifications](#run-local-verifications)
+- [The Pull Request Submit Process](#the-pr-submit-process)
+  * [The Testing and Merge Workflow](#the-testing-and-merge-workflow)
+  * [Marking Unfinished Pull Requests](#marking-unfinished-pull-requests)
+  * [Comment Commands Reference](#comment-commands-reference)
+  * [Automation](#automation)
+  * [How the e2e Tests Work](#how-the-e2e-tests-work)
+- [Why was my pull request closed?](#why-was-my-pr-closed)
+- [Why is my pull request not getting reviewed?](#why-is-my-pr-not-getting-reviewed)
```
Comment (tpepper) [inline contributors/guide/pull-requests.md]: for these two section title lines suggest Pull Request...with caps to be consistent with the other changes

File: contributors/guide/pull-requests.md [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+- [Best Practices for Faster Reviews](#best-practices-for-faster-reviews)
+  * [0. Familiarize yourself with project conventions](#0-familiarize-yourself-with-project-conventions)
+  * [1. Is the feature wanted? Make a Design Doc or Sketch pull request](#1-is-the-feature-wanted-make-a-design-doc-or-sketch-pr)
+  * [2. Smaller Is Better: Small Commits, Small pull requests](#2-smaller-is-better-small-commits-small-prs)
```
Comment (tpepper) [inline contributors/guide/pull-requests.md]: these two also

File: contributors/guide/pull-requests.md [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+
+- [Sign the CLA](https://git.k8s.io/community/CLA.md) (prerequisite)
+- [Open a pull request](https://help.github.com/articles/about-pull-requests/)
+  - *For kubernetes/kubernetes repository only:* Add [release notes](/guide/release-notes.md) if needed.
+- Pass all e2e tests
+- Get all necessary approvals from reviewers and code owners 
+
+## The Testing and Merge Workflow
+
+The Kubernetes merge workflow uses comments to run tests and labels for merging pull requests.
```
Comment (tpepper) [inline contributors/guide/pull-requests.md]: What do you think about something like:

"...uses special labels (link?) and commands(link https://prow.k8s.io/command-help) to trigger actions leading to merging pull requests."

Authoritative for labels might be https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.yaml but that's not exactly human readable content.

Comment (tpepper) [reply contributors/guide/pull-requests.md]: This labels and commands topic probably merits its own section.  That would help the TODO a few lines below.

File: contributors/guide/pull-requests.md [hunk 5]
```diff
@@ -0,0 +1,283 @@
...
+1. Push edits to your pull request branch
+1. Repeat the prior two steps as needed until reviewer(s) add `/lgtm` label
+1. (Optional) Some reviewers prefer that you squash commits at this step
+1. Follow the bot suggestions to assign an OWNER who will add the `/approve` label to the pull request
+
+<!-- TODO: explain about lgtm and approve labels; possibly link to /devel/owners.md -->
+Once the tests pass, all failures are commented as flakes, or the reviewer adds the labels `/lgtm` and `/approved`, the pull request enters the final merge queue. The merge queue is needed to make sure no incompatible changes have been introduced by other pull requests since the tests were last run on your pull request.
+<!-- TODO: create parallel instructions for reviewers -->
+
+Either the [on call contributor](/contributors/devel/on-call-rotations.md) will manage the merge queue manually, or the [GitHub "munger"](https://git.k8s.io/test-infra/mungegithub) submit-queue plugin will manage the merge queue automatically.
```
Comment (tpepper) [inline contributors/guide/pull-requests.md]: on-call-rotations.md went away at commit fc975ed59114079b3a3789fed4c2d3aac0b3cda3

File: contributors/guide/pull-requests.md [hunk 6]
```diff
@@ -0,0 +1,283 @@
...
+Let's talk about best practices so your pull request gets reviewed quickly.
+
+## 0. Familiarize yourself with project conventions
+
+* [Development guide](/contributors/devel/development.md)
+* [Coding conventions](../guide/coding-conventions.md)
+* [API conventions](/contributors/devel/api-conventions.md)
+* [Kubectl conventions](/contributors/devel/kubectl-conventions.md)
+
+## 1. Is the feature wanted? Make a Design Doc or Sketch pull request
```
Comment (tpepper) [inline contributors/guide/pull-requests.md]: Reference KEP and feature processes.

Looks like a lot has changed since this section was written.

**Comment by tpepper** [comment]
/hold

I'd like to help @guineveresaenger swizzle these commits around so that the commit history will be more clear, even if it's all squashed.

**Comment by tpepper** [comment]
/unhold

These new commits are nicer with the git mv and changes so the history is kept and nice split of logical changes for easier review!

File: contributors/devel/faster_reviews.md [hunk 7]
```diff
@@ -1,3 +1,3 @@
 # How to get faster PR reviews
 
-Merged with the [pull requests doc](pull-requests.md#best-practices-for-faster-reviews).
+Merged with the [pull requests doc](/contributors/guide/pull-requests.md#best-practices-for-faster-reviews).
```
Comment (cblecker) [inline contributors/devel/faster_reviews.md]: Please add a tombstone to this file with an end date for removal.

File: contributors/guide/release-notes.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
+On the kubernetes/kubernetes repository, release notes are required for any pull request with user-visible changes, such as bug-fixes, feature additions, and output format changes.
```
Comment (cblecker) [inline contributors/guide/release-notes.md]: This doc needs a title

File: contributors/guide/release-notes.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+For pull requests that don't need to be mentioned at release time, just write "NONE" (case insensitive):
+
+    ```release-note
+    NONE
+    ```
+
+The `/release-note-none` comment command can still be used as an alternative to writing "NONE" in the release-note block if it is left empty.
+
+To see how to format your release notes, view the kubernetes/kubernetes[pull request template](https://git.k8s.io/kubernetes/.github/PULL_REQUEST_TEMPLATE.md) for a brief example. pull request titles and body comments can be modified at any time prior to the release to make them friendly for release notes.
```
Comment (cblecker) [inline contributors/guide/release-notes.md]: nit: missing space `kubernetes/kubernetes[pull request template]`

File: contributors/guide/pull-requests.md [hunk 10]
```diff
@@ -0,0 +1,280 @@
...
+If you want to solicit reviews before the implementation of your pull request is complete, you should hold your pull request to ensure that the merge queue does not pick it up and attempt to merge it. There are two methods to achieve this:
+
+1. You may add the `/hold` or `/hold cancel` comment commands
+2. You may add or remove a `WIP` or `[WIP]` prefix to your pull request title
+
+The GitHub robots will add and remove the `do-not-merge/hold` label as you use the comment commands and the `do-not-merge/work-in-progress` label as you edit your title. While either label is present, your pull request will not be considered for merging.
+
+## Comment Commands Reference
+
+[The commands doc](https://prow.k8s.io/command-help.html) contains a reference for all comment commands.
```
Comment (cblecker) [inline contributors/guide/pull-requests.md]: https://go.k8s.io/bot-commands

File: contributors/devel/faster_reviews.md [hunk 11]
```diff
@@ -1,3 +1,4 @@
-# How to get faster PR reviews
-
-Merged with the [pull requests doc](/contributors/guide/pull-requests.md#best-practices-for-faster-reviews).
+The contents of this file have been moved to  https://git.k8s.io/community/contributors/guide/pull-requests.md.
+ <!--
+ This file is a placeholder to preserve links.  Please remove after 3 months or the release of kubernetes 1.10, whichever comes first.
```
Comment (tpepper) [inline contributors/devel/faster_reviews.md]: I'm still unclear on the policy for how long to keep these tombstones, but as we get closer to 1.10 release, we probably should think about changing 3 mo's or 1.10 to instead be 3 mo's or 1.11.

**Comment by guineveresaenger** [comment]
@tpepper fixed commit message
